### PR TITLE
Database Tests

### LINKED ISSUE #40: Database Tests
I will create database unit tests so that everything works thoroughly.

### PR DESCRIPTION
**Pull request by EricKarpovits**
I've added 7 Database tests that cover all the methods in the Gateway. 

The things that I test for:
- Create;Add an object; Save; Load 
- Remove an object from the database and test that it also saved the removal when loading in
- Test adding multiple objects and multiple keys
- Test saving and loading an empty Database
- Simple test on the hasKey method in the gateway
- Test the deleteStorageFile method and that it successfully deletes the storage file
- Test that the constructor successfully creates a valid storage file that will persist - even when the computer is turned off. 

Let me know if there is something I should change :).

### CODE REVIEW

File: src/main/java/database/StorageObjects.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package database;
 
+import entities.fridge.CommonFridge;
```
Comment (rickychen175) [inline src/main/java/database/StorageObjects.java:3]: Is this import being used?

**Review by rickychen175 — APPROVED** [review]

**Review by Shaffaan — APPROVED** [review]
Really good work eric, especially checking all the edge cases!

**Review by Zhuofan-Y — APPROVED** [review]
Nice tests Eric!

File: src/main/java/database/StorageObjects.java (hunk 1 above)
Comment (EricKarpovits) [reply src/main/java/database/StorageObjects.java:3]: to initialize the fridge.
